blas/testblas/ztbsv.go

```diff
@@ -0,0 +1,105 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testblas
+
+import (
+	"fmt"
+	"testing"
+
+	"golang.org/x/exp/rand"
+	"gonum.org/v1/gonum/blas"
+)
+
+type Ztbsver interface {
+	Ztbsv(uplo blas.Uplo, trans blas.Transpose, diag blas.Diag, n, k int, ab []complex128, ldab int, x []complex128, incX int)
+
+	Ztrmver
+}
+
+func ZtbsvTest(t *testing.T, impl Ztbsver) {
+	rnd := rand.New(rand.NewSource(1))
+	for _, uplo := range []blas.Uplo{blas.Upper, blas.Lower} {
+		for _, trans := range []blas.Transpose{blas.NoTrans, blas.Trans, blas.ConjTrans} {
+			for _, diag := range []blas.Diag{blas.NonUnit, blas.Unit} {
+				for _, n := range []int{1, 2, 3, 4, 10} {
+					for k := 0; k < n; k++ {
+						for _, ldab := range []int{k + 1, k + 1 + 10} {
+							for _, incX := range []int{-4, 1, 5} {
+								name := fmt.Sprintf("uplo=%v,trans=%v,diag=%v,n=%v,k=%v,ldab=%v,incX=%v", uplo, trans, diag, n, k, ldab, incX)
+								t.Run(name, func(t *testing.T) {
```

Comment:
I'm not convinced that this is a good idea; it results in spamming many (1440) lines to the logs on travis (we use `-v`, so all the subtests get logged even when they pass). If we do this for all tests, we run out of our log limit.

Reply:
Ok, I'll change it to the old style here and in Ztbmv.